Postman/Postman.js

```diff
@@ -58,27 +53,27 @@ Postman.prototype.sent = function(done){
 @param { FUnction } done
 @return { Error | Array }
 **/
-Postman.prototype.fetchSeries = function(user, done){
-	var _self = this;
-	var conditions = {
-		$and: [
-			{
-				_id: {
-					$in: user.seriesList.map(function(userSeriesItem){
-						return userSeriesItem._id;
-					})
-				}
-			},
-			{ hasNewRelease: true }
-		]
-	};
-
-	this.Series.find(conditions, function(err, seriesItems){
-		if(err){
-			return done(err);
-		}
-		done(null, _self._filterSeries(user, seriesItems));
-	});
+Postman.prototype.fetchSeries = function fetchSeries(user, done) {
+  const _self = this;
+  const conditions = {
+    $and: [
+      {
+        _id: {
+          $in: user.seriesList.map((userSeriesItem)=> {
+            return userSeriesItem._id;
+          }),
+        },
+      },
+      { hasNewRelease: true },
+    ],
+  };
+
+  this.Series.find(conditions, (err, seriesItems)=> {
```

Comment:
__[jshint]__
*  'arrow function syntax (=>)' is only available in ES6 (use esnext option).